geometry_msgs/Pose[] cartesian_wp
---
std_msgs/Bool success
